### PR TITLE
Authenticate with Passkey

### PR DESCRIPTION
**Pull request by turnipdabeets**
- [ ] wait for #207 to merge (need to update links)
- [ ] add "Our view on using industry identity standards for authentication" - I'm not actually sure what that is

Click on the `View Deployment` button below to see the web page generated with these changes

### CODE REVIEW

**Review by Will5 — APPROVED** [review]
Looks cool!

File: docs/workflows/authentication.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TokenExchange from './\_authentication-token-exchange.mdx'
+import WebAuth from './\_authentication-via-automatic.mdx'
+import SilentAuth from './\_authentication-via-return.mdx'
+
+# Authentication Overview
+
+This guide describes how to authenticate an application with a passkey using an OAuth2 authorization flow. This guide assumes you have already [set up an application](../using-bi-for-auth.md), have access to the Beyond Identity Admin Console, and your application has at least one passkey [bound to an identity](./TODO link ./bind-passkey). Before authenticating, some logic should be performed to select a passkey. This might be UI presented to the user to select a passkey. Once a passkey is selected, you can authenticate with that passkey. There are two steps to authentication: [authorization](authentication#authorization) and then [token exchange](authentication#token-exchange). First you will need to authorize the application to recieve authorization code. That code can be then used in the token exchange.
```
Comment (sublimezeal) [inline docs/workflows/authentication.md:14]: Suggest a slight re-wording to emphasize how we fit into a standard OAuth/OIDC flow:

This guide describes how to authenticate an application with a passkey using Beyond Identity within a standard OAuth2/OIDC authorization flow. 

In response to an OIDC request to the Beyond Identity `/authorize` endpoint, Beyond Identity initiates passwordless authentication by returning an authentication challenge and other information to your app. Before authenticating, your app can use the Beyond Identity SDK to enumerate available passkeys and should perform some logic to select one, such as presenting selection UI to the user. Once a passkey is selected, you can then use the SDK to complete authentication and finally perform the OAuth code for token exchange. 

This guide assumes you have already set up an application, have access to the Beyond Identity Admin Console, and your application has at least one passkey [bound to an identity](./TODO link ./bind-passkey).

**Review by sublimezeal — APPROVED** [review]
some comments, and otherwise i approve this new doc

**Review by ismailmustafa — APPROVED** [review]

**Comment by beyondchojnacki** [comment]
Looks great!

**Pull request merged**